packages/neon/neon/lib/src/models/account.dart

```diff
@@ -72,11 +72,9 @@ class Account implements Credentials {
     return _idCache[key] = sha1.convert(utf8.encode(key)).toString();
   }
 
-  String get humanReadableID {
-    final uri = Uri.parse(serverURL);
-    // Maybe also show path if it is not '/' ?
-    return '$username@${uri.port != 443 ? '${uri.host}:${uri.port}' : uri.host}';
-  }
+  // Maybe also show path if it is not '/' ?
+  String get humanReadableID =>
+      '$username@${serverURL.port != 443 ? '${serverURL.host}:${serverURL.port}' : serverURL.host}';
```

Comment:
```suggestion
      '$username@${serverURL.host}${serverURL.port != 443 ? ':${serverURL.port}' : ''}';
```

How about also using a string buffer for this? I think that would be a lot cleaner.